packages/api/tests/test-es-indexer.js

```diff
@@ -383,6 +383,52 @@ test.serial('indexing a collection record', async (t) => {
   t.is(typeof record._source.timestamp, 'number');
 });
 
+test.serial('updating a collection record', async (t) => {
```

Comment:
perhaps we could also include a similar test but where the second collection object has a different version - asserting that 2 different records get created?

Reply:
added test